File: 1 - INICIANTE/C++17/1043.cpp
#include <iostream>
#include <iomanip>
#include <cmath>
using namespace std;

int main() {
  double a,b,c,x,y;

  cin >> a >> b >> c;

  x = a + b + c; // Perimetro Triangulo
  y = ((a+b)*c)/2; // Area Trapezio

  if ((abs(b-c)<a && a<(b+c)) || (abs(a-c)<b && b<(a+c)) || (abs(a-b)<c && c<(a+b))) {
    cout << fixed << setprecision(1) << "Perimetro = " << x << "\n";
  }
  else {
    cout << fixed << setprecision(1) << "Area = " << y << "\n";
  }
return 0;
}

File: 1 - INICIANTE/python/1043.py
value = str(input()).split(' ')
a, b, c = float(value[0]), float(value[1]), float(value[2])

if (a+b>c) and (a+c>b) and (b+c>a):
    print('Perimetro = {:.1f}'.format((a+b+c)))
else:
    print('Area = {:.1f}'.format(((a+b)*c/2)))

